src/pages/ProductPage/index.jsx

```diff
@@ -1,28 +1,36 @@
-import { useEffect, useState } from "react";
+import { useEffect, useState, useContext } from "react";
 import ProdDetail from "./ProdDetail/ProdDetail";
 import ProdSelector from "./ProdSelector/ProdSelector";
 import Gallery from "../../components/Gallery/Gallery";
+import CartContext from "../../components/TopBarMenu/Cart/CartContext.js";
 
 import "./index.scss";
 
 export default function ProductPage() {
-  const [products, setProduct] = useState([]);
+  const [product, setProduct] = useState(undefined);
+  const { cart, setCart } = useContext(CartContext);
 
   useEffect(() => {
     fetch("/data.json")
       .then((response) => response.json())
       .then((data) => {

```

Comment:
You can change the name to be more meanful:

```suggestion
      .then((productData) => {
```